packages/squiggle-lang/__tests__/Reducer/Reducer_functionAssignment_test.res

```diff
@@ -0,0 +1,12 @@
+open Jest
+open Reducer_TestHelpers
+
+describe("Parse function assignment", () => {
+  testParseToBe("f(x)=x", "Ok((:$let :f (:$lambda (x) :x)))")
+  testParseToBe("f(x)=2*x", "Ok((:$let :f (:$lambda (x) (:multiply 2 :x))))")
+  //MathJs does not allow blocks in function definitions
+})
+
+describe("Evaluate function assignment", () => {
```

Comment:
`describe` string should be `Assign then evaluate identity function`

Reply:
More to come. That description is not only for that expression...